popup/default.html

```diff
@@ -3,17 +3,31 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <h1>Praise editor</h1>
-    <p>Enter or edit some praise markdown below.</p>
+    <h1>You're Doing A Good Job!</h1>
+    <h2>Configuration</h2>
     <form>
-      <p>
-        <label for="praiseMarkdown">Praise Markdown</label>
-        <textarea name="praiseMarkdown" id="praiseMarkdown"></textarea>
-      </p>
-      <p>
-        <button type="reset">Reset</button>
+      <div>
         <button type="submit">Update</button>
-      </p>
+      </div>
+      <fieldset>
+        <legend>Unsplash API key</legend>
```

Comment:
todo: move api key to last in the form, or hide it behind a details expandy, because this will be set once and forgotten.

Reply:
nahhh, I'm rejecting this feedback. Adding a details/summary expandy looks funny, and the field is critical to functionality so I don't want to move it to a place it might not be found.